.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,71 @@
+name: Bug Report
+description: I experienced unexpected behavior using useSimpleReducer.
```

Comment:
I would remove anything project-specific from these templates so we can just copy-paset them between projects.